src/lama_aesthetics/plotutils.py

```
@@ -105,3 +119,201 @@ def callback(axes):
+    target_ax.set_title(source_ax.get_title())
+    target_ax.set_xscale(source_ax.get_xscale())
+    target_ax.set_yscale(source_ax.get_yscale())
+
+    # Copy tick parameters
+    target_ax.set_xticks(source_ax.get_xticks())
+    target_ax.set_yticks(source_ax.get_yticks())
+    try:
+        target_ax.set_xticklabels([t.get_text() for t in source_ax.get_xticklabels()])
+        target_ax.set_yticklabels([t.get_text() for t in source_ax.get_yticklabels()])
+    except Exception:
+        pass
```

Ruff (0.15.5)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)